test/clojure-mode-indentation-test.el

```diff
@@ -68,9 +68,11 @@ values of customisable variables."
            (search-forward "|")
            (delete-char -1)
            (clojure-mode)
+           (font-lock-ensure)
            (indent-according-to-mode)
 
-           (should (equal expected-state (buffer-string)))
+           (should (equal expected-state (buffer-substring-no-properties
```

Comment:
Why is this change necessary?

Reply:
Because `clojure-in-docstring-p` checks for a font-lock-face, `indent-according-to-mode` works differently (in this case) if font-lock-mode is off, so I wrote `font-lock-ensure` since otherwise there is no font-lock information in the temp-buffer.

If this change wasn't there, this test: 

https://github.com/clojure-emacs/clojure-mode/blob/43e3e65966b8798968016bf0134a726c4a73461e/test/clojure-mode-indentation-test.el#L123-L125

would always pass, both in the current version and my version. To make it fail in the current version (which is expected and wanted), the change had to be made. I found no other way to mimic what is really happening.

But you are right, I'll compare the strings without removing the properties, no reason to do that.